src/lib/service-bus/FFC-Message-Sender.js

```diff
@@ -0,0 +1,26 @@
+const { MessageSender } = require('ffc-messaging');
+
+/**
+ *
+ */
+export class AzureServiceBus {
```

Comment:
Is this file needed? The file name and the class name don't match. Maybe I'm missing something.